cognite/client/_api/datapoints.py

```diff
@@ -592,6 +595,7 @@ def __init__(self, config: ClientConfig, api_version: Optional[str], cognite_cli
         self._DPS_INSERT_LIMIT = 100_000
         self._RETRIEVE_LATEST_LIMIT = 100
         self._POST_DPS_OBJECTS_LIMIT = 10_000
+        self._GRANULARITY_HOURS_LIMIT = 100_000.0
```

Comment:
...float? 🤔

Reply:
just to avoid mypy bugging me, but can make it into an int